src/_includes/banner.html

```diff
@@ -2,8 +2,9 @@
   For headings use h3 elements.
 {% endcomment -%}
 <div class="site-banner site-banner--default" role="alert">
-  Flutter 3.22 is live! Check out the <a href="https://medium.com/flutter/io24-5e211f708a37">Flutter 3.22 and Dart 3.4</a> announcement,<br>
+  <p>Flutter 3.22 is live! Check out the <a href="https://medium.com/flutter/io24-5e211f708a37">Flutter 3.22 and Dart 3.4</a> announcement,<br>
   the <a href="https://medium.com/flutter/whats-new-in-flutter-3-22-fbde6c164fe3">3.22 technical</a> blog post.<br>
-  Also check out <a href="/release/whats-new">what's new</a> on the website.
+  Also check out <a href="/release/whats-new">what's new</a> on the website.</p>
+  <p>Help improve Flutter! Take our <a href="https://google.qualtrics.com/jfe/form/SV_2mXOpGTNRLBQLWe?Source=Website">Q2 survey</a>.</p>
```

Comment:
**suggestion:** This creates a huge banner. Let's trim it back some.

```suggestion
<a href="https://medium.com/flutter/io24-5e211f708a37">Flutter 3.22 and Dart 3.4</a> are live! Check out <a href="/release/whats-new">what's new</a> on the website.<br>
Help improve Flutter!<br>
Take our <a href="https://google.qualtrics.com/jfe/form/SV_2mXOpGTNRLBQLWe?Source=Website">Q2 survey</a>.
```

Reply:
Thanks! I added a commit to reflect this change.